spec.md

```diff
@@ -2836,6 +2997,7 @@ type(x) returns a string describing the type of its operand.
 ```python
 type(None)              # "NoneType"
 type(0)                 # "int"
+type(0.0)               # "float"
 ```
 
 ### zip
```

Comment:
The section on Grammar reference at the bottom needs updating. Likewise for tokens.

Reply:
Done.